internal/app/cache/cache.go

```diff
@@ -233,7 +233,7 @@ func (c *cache) DeleteRequest(key string, req transport.Request) error {
 	c.cache.Add(key, resource)
 	c.logger.With(
 		"aggregated_key", key,
-		"node_id", req.GetNodeID,
+		"node_id", req.GetNodeID(),
```

Comment:
wow. What does this even print if it's not a function call? The address of the function?

Reply:
it printed the function name  😄